osu.Game/Overlays/FirstRunSetupOverlay.cs

```diff
@@ -0,0 +1,397 @@
+            new FirstRunStep(typeof(ScreenUIScale), GraphicsSettingsStrings.UIScaling),
+        };
+
+        private Container stackContainer = null!;
+
+        private Bindable<OverlayActivation>? overlayActivationMode;
+
+        public FirstRunSetupOverlay()
+        {
+            RelativeSizeAxes = Axes.Both;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Anchor = Anchor.Centre;
+            Origin = Anchor.Centre;
+
+            RelativeSizeAxes = Axes.Both;
+            Size = new Vector2(0.95f);
+
+            EdgeEffect = new EdgeEffectParameters
+            {
+                Type = EdgeEffectType.Shadow,
+                Radius = 5,
+                Colour = Color4.Black.Opacity(0.2f),
+            };
+
+            Masking = true;
+            CornerRadius = 10;
+
+            Children = new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colourProvider.Background6,
+                },
+                new GridContainer
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    RowDimensions = new[]
+                    {
+                        new Dimension(GridSizeMode.AutoSize),
+                        new Dimension(),
+                        new Dimension(GridSizeMode.AutoSize),
+                    },
+                    Content = new[]
+                    {
+                        new Drawable[]
+                        {
+                            new Container
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Children = new Drawable[]
+                                {
+                                    new Box
+                                    {
+                                        Colour = colourProvider.Background5,
+                                        RelativeSizeAxes = Axes.Both,
+                                    },
+                                    new FillFlowContainer
+                                    {
+                                        RelativeSizeAxes = Axes.X,
+                                        Margin = new MarginPadding(10),
+                                        AutoSizeAxes = Axes.Y,
+                                        Direction = FillDirection.Vertical,
+                                        Children = new Drawable[]
+                                        {
+                                            new OsuSpriteText
+                                            {
+                                                Text = FirstRunSetupOverlayStrings.FirstRunSetup,
+                                                Font = OsuFont.Default.With(size: 32),
+                                                Colour = colourProvider.Content2,
```

Comment:
Bit of a minor thing but the colour choices here look a bit backwards to me. [The colour guide](https://www.figma.com/file/VIkXMYNPMtQem2RJg9k2iQ/Asset%2FColours?node-id=1837%3A2) says that primary content should use `Content1` and secondary content should use `Content2`, but here it seems to be the other way around.

Compare the settings panel - the main heading there is `Content1` and the subheading is `Content2`.